package.json

```diff
@@ -89,7 +89,7 @@
     "express": "4.17.1",
     "fake-indexeddb": "3.1.3",
     "filehound": "1.17.5",
-    "full-icu": "1.3.4",
+    "full-icu": "1.4.0",
```

Comment:
Please apply at least this part. It unbreaks building Brave on Redhat/CentOS/Fedora. See https://github.com/nodejs/full-icu-npm/issues/63